Strings/MergeTools

```diff
@@ -0,0 +1,17 @@
+'''
+Title     : Merge The Tools!
+Subdomain : Strings
+Domain    : Python
+Author    : Atharva Shah
+Created   : 15 March 2021
+Problem   : https://www.hackerrank.com/challenges/merge-the-tools/
+'''
+from collections import OrderedDict
+
+def merge_the_tools(string, k):
+    for i in range(0, len(string), k):
```

Comment:
Your solution passes all the cases. But I think we can change this line to: `for i in range(0, len(string)-k+1, k):`

Reply:
Done. Yes, this is certainly better.